src/messages/messages.json

```diff
@@ -1,7 +1,13 @@
 {
-    "run_started": "Running SOAtest in ",
-    "run_failed": "Failed to run SOAtest",
-    "exit_code": "SOAtest run finished with code ",
-    "failed_run_non_zero": "SOAtest run failed with a non-zero exit code: ",
-    "wrk_dir_not_exist": "Working directory does not exist: "
+    "run_started": "Running SOAtest in {0}",
+    "run_failed": "Run failed",
+    "exit_code": "SOAtest run finished with code {0}",
+    "failed_run_non_zero": "SOAtest run failed with a non-zero exit code: {0}",
+    "wrk_dir_not_exist": "Working directory does not exist: {0}",
+    "can_not_process_soatest_report": "Can not process Parasoft SOAtest report due to the XML report can not be found related to report input value: {0}",
```

Comment:
Please change to: Parasoft SOAtest report not found at the specified location: {0}